cmd/award.go

```diff
@@ -49,7 +48,8 @@ to quickly create a Cobra application.`,
 
 func init() {
 	rootCmd.AddCommand(awardCmd)
-	awardCmd.PersistentFlags().StringVarP(&message, "message", "m", "You're awesome!", "Your thank you/appreciation message that will be visible to everone.")
+	awardCmd.PersistentFlags().StringVarP(&message, "message", "m", "You're awesome!", `Your thank you/appreciation message that will be visible
```

Comment:
Any specific reason why you changed `"` to `` ` ``?
And why did you insert the newline? I think the line length is fine, especially considering this is just a help message that is not helpful during development, i.e. it's not a problem if it is off-screen.

Reply:
Line seems too long. What do you mean by during development? How do you think is the proper way to address this?

Reply:
Ah, I think I know what you mean now. The help messages when typing `bonusly award --help` were too long before.
I just checked - looks good with the linebreak 👍🏼